perf/load/values.yaml

```diff
@@ -13,6 +13,10 @@ readinessProbe:
   initialDelaySeconds: 5
   periodSeconds: 5
 
+# request cpu/memory resources of istio-proxy
+proxyCPU: 100m
```

Comment:
should we keep defaults the same as standard Istio install? Or even leave these blank and only set annotation if they have overriden the helm value?

Reply:
Hi @howardjohn ,in standard istio install, what is the default request of cpu and memory? I thought it is 100m and 128Mi.

Reply:
Oh you are right. Thought it was 200m